academy/4-my-own-chain/game-wager.md

```diff
@@ -674,8 +675,8 @@ You learned in a [previous section](./stored-game.md) how to launch your test in
 
 ![Suite runner with green button](/go_test_debug_suite.png)
 
-Note that you can only launch debug for all of the suite's test methods, and not just a single one as is possible with a simple test. For this reason, the case could be made that you could create more granular suites. For example, one or more test suites per file.
+Note that you can only launch debug for all of the suite's test methods and not just a single one (as is possible with a simple test). A sollution to this is to create more granular suites, for example using one or more test suites per file.
```

Comment:
```suggestion
Note that you can only launch debug for all of the suite's test methods and not just a single one (as is possible with a simple test). A solution to this is to create more granular suites, for example using one or more test suites per file.
```
@AndrewB9lab

Reply:
Thanks - I really need to get to grips with my keyboard better, I keep double-tapping.